device/iot-device-client/pom.xml

```diff
@@ -36,19 +36,19 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.8</version>
+            <version>${lombok-version}</version>
```

Comment:
I like making these variables; however I would favor using the <dependencyManagement> method over setting a build property to specify the needed dependency versions in the parent pom. We don't utilize the properties anyway.

Take a look at https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-management